config/webpack.dev.js

```diff
@@ -52,7 +52,18 @@ module.exports = function (options) {
     plugins: [
       new DashboardPlugin(),
       extractCSS,
-      extractSASS,
+      // extractSASS,
+      /*
+       * StyleLintPlugin
+       */
+      new StyleLintPlugin({
```

Comment:
Is this defined in this file? I see one in webpack.common.js but there is no reference of that plugin here

Reply:
Definitely a mistake. I've added the definition to this file.